GeneralsMD/Code/Libraries/Source/WWVegas/WW3D2/lightenvironment.cpp

```diff
@@ -52,8 +52,10 @@ const float DIFFUSE_TO_AMBIENT_FRACTION = 1.0f;
 /*
 ** Static variables
 */
-static _LightingLODCutoff			= 0.5f;	
-static _LightingLODCutoff2			= 0.5f * 0.5f;
+// TheSuperHackers @fix xezon 13/03/2025 Set integer type as per the original.
+// TODO: Investigate if it should be a float.
+static int _LightingLODCutoff			= 0.5f;
```

Comment:
This was definitely intended to be float. This also affects the Renegade code and is fixed in the Linux Dedicated Server build (presumably because GCC was more standard compliant at the time and errored out trying to build it). What if anything it affects in Generals and if we want to fixed it correctly I couldn't say.